sdk/avalon_sdk/ethereum/ethereum_worker_registry_list.py

```diff
@@ -245,15 +245,15 @@ def registry_update(self, org_id, uri, sc_addr, app_type_ids):
                     logging.error("Invalid application id {}".format(aid))
                     return None
 
-            txn_statusn_hash = \
+            txn_dict = \
                 self.__contract_instance.functions.registryUpdate(
                     org_id, uri, sc_addr,
                     app_type_ids).buildTransaction(
                         self.__eth_client.get_transaction_params()
                         )
-            txn_status = self.__eth_client.execute_transaction(
-                txn_statusn_hash)
-            return txn_status
+            txn_receipt = self.__eth_client.execute_transaction(
+                txn_dict)
+            return txn_receipt
```

Comment:
function comments mentions Returns None on error. Does txn_receipt corresponds to error code ?